deploy/nginx/nginx.conf

```diff
@@ -52,7 +52,65 @@ http {
+        rewrite ^/scheduler/kubernetes/logmanagement-efk-k8s.html /portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/monitoring/logmanagement-efk-k8s permanent;
+        rewrite ^/scheduler/kubernetes/monitoring-px-prometheusAndGrafana.html /portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/monitoring/monitoring-px-prometheusandgrafana.1 permanent;
+        rewrite ^/scheduler/kubernetes/non-root.html /portworx-install-with-kubernetes/storage-operations/create-pvcs/access-via-non-root-users permanent;
+        rewrite ^/scheduler/kubernetes/openshift-install.html /portworx-install-with-kubernetes/on-premise/openshift permanent;
+        rewrite ^/scheduler/kubernetes/preprovisioned-volumes.html /portworx-install-with-kubernetes/storage-operations/create-pvcs/using-preprovisioned-volumes permanent;
+        rewrite ^/scheduler/kubernetes/pvc-encryption.html /portworx-install-with-kubernetes/storage-operations/create-pvcs/create-encrypted-pvcs permanent;
+        rewrite ^/scheduler/kubernetes/px-k8s-asg.html /portworx-install-with-kubernetes/cloud/aws/aws-asg permanent;
+        rewrite ^/scheduler/kubernetes/px-k8s-interactive.html /interactive-tutorials permanent;
+        rewrite ^/scheduler/kubernetes/px-k8s-spec-curl.html /portworx-install-with-kubernetes/px-k8s-spec-curl permanent;
+        rewrite ^/scheduler/kubernetes/resize-pvc.html /portworx-install-with-kubernetes/storage-operations/create-pvcs/resize-pvc permanent;
+        rewrite ^/scheduler/kubernetes/scheduler-convergence-labels.html /portworx-install-with-kubernetes/storage-operations/hyperconvergence permanent;
+        rewrite ^/scheduler/kubernetes/scheduler-convergence.html /portworx-install-with-kubernetes/storage-operations/hyperconvergence permanent;
+        rewrite ^/scheduler/kubernetes/set-cluster-wide-secret.html /reference/cli/encrypted-volumes permanent;
+        rewrite ^/scheduler/kubernetes/shared-volumes.html /portworx-install-with-kubernetes/storage-operations/create-pvcs/create-shared-pvcs permanent;
+        rewrite ^/scheduler/kubernetes/snaps-3d.html /portworx-install-with-kubernetes/storage-operations/create-snapshots/snaps-3d permanent;
+        rewrite ^/scheduler/kubernetes/snaps-annotations.html /portworx-install-with-kubernetes/storage-operations/create-snapshots permanent;
+        rewrite ^/scheduler/kubernetes/snaps-cloud.html /portworx-install-with-kubernetes/storage-operations/create-snapshots/snaps-cloud permanent;
+        rewrite ^/scheduler/kubernetes/snaps-group.html /portworx-install-with-kubernetes/storage-operations/create-snapshots/snaps-group permanent;
+        rewrite ^/scheduler/kubernetes/snaps-local.html /portworx-install-with-kubernetes/storage-operations/create-snapshots/snaps-local permanent;
+        rewrite ^/scheduler/kubernetes/snaps-single-pvc.html /portworx-install-with-kubernetes/storage-operations/create-pvcs permanent;
```

Comment:
/scheduler/kubernetes/snaps-single-pvc.html to /portworx-install-with-kubernetes/storage-operations/create-snapshots/snaps-single-pvc/